README.md

```diff
@@ -20,4 +20,4 @@ Add your answers to this markdown file.
 
 2. The time compelexity could only be expected to be log(10000) if the tree is balanced, but there is no way to know the exact time it would take to find the same element. It would depend on many different factors such as hardware and the data that was put into the tree. 
 
-3. One reason this could have happened is because the tree is unbalanced, which causes the computer to do more work in search of the element. Another reason may be that the size of the array is too big for one memory allocation, so the computer has to use a larger but slower memory unit to store the tree. A third reason this could happen is because the search function for the element in the tree is written enefficiently and calls unnecessary function calls or iterations. 
+3. One reason this could have happened is because the hardware of the computer executing the code may handle the size of this tree poorly. Another reason may be that the function that times the run time of the binary search could be faulty. A third reason this could happen is because the search function for the element in the tree is written enefficiently and calls unnecessary function calls or iterations.
```

Comment:
What exactly do you mean when you say that the hardware can handle it poorly and that the search could be faulty?

Reply:
The hardware of the computer could have a slow cpu that processes the search very slow and a slow hard drive that struggles to read and write from memory and increasing the size of the tree could affect them more and more. The other part was I was saying that the timing function that tracks the speed of the search function could be wrong. So the time it displays for the run time of the search function is not actually correct.

Reply:
Well, let's assume here that the timing is correct. And the hardware would affect the smaller runtime as well, unless you're saying that it's more than a constant factor? Why would it be more than a constant factor?

Reply:
Maybe the hard drive copies the data at each step of recursion so that the memory stored is increased exponentially.

Reply:
But you're not using exponentially more memory.